knowledge-graph/src/main/scala/io/renku/knowledgegraph/MicroserviceRoutes.scala

```diff
@@ -126,6 +127,7 @@ private class MicroserviceRoutes[F[_]: MonadThrow](
   private lazy val nonAuthorizedRoutes: HttpRoutes[F] = HttpRoutes.of[F] {
     case GET -> Root / "knowledge-graph" / "graphql" => schema()
     case GET -> Root / "ping"                        => Ok("pong")
+    case GET -> Root / "spec.json"                   => docsEndpoint.`get /docs`
```

Comment:
Should we maybe choose a name for the API that is more in line with the other APIs? Perhaps, something like `api-spec`?

Reply:
Once you decide on the API could you adapt both the name of the endpoint and the `def` on it, please?